packages/foreman/rubygem-autoprefixer-rails/rubygem-autoprefixer-rails.spec

```diff
@@ -1,54 +1,44 @@
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: default
 %global gem_name autoprefixer-rails
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 5.2.1.3
-Release: 7%{?dist}
-Summary: Add vendor prefixes to CSS rules using values from caniuse.com
-Group: Development/Languages
+Name: rubygem-%{gem_name}
+Version: 10.4.7.0
```

Comment:
This is a huge jump, but it was mostly to drop old versions of NodeJS. We shouldn't be affected (and Debian probably already uses this anyway).